lib/rmt.rb

```diff
@@ -1,5 +1,5 @@
 module RMT
-  VERSION ||= '2.15'.freeze
+  VERSION ||= '2.15-pre'.freeze
```

Comment:
Why are you using the `-pre` version specifier?

Reply:
I was testing something and pushed it accidentally. I removed it!

Reply:
Good catch, I thought that with our shiny fast CI we were doing pre-releases ;)